README.md

```diff
@@ -1,2 +1,88 @@
-# online-bookstore
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# online-bookstore app
 If you are keen to reading, you probably forget the books you have read or where you were reaching while reading a particular book. This app is here to help you overcome that. It is a book store management system where you can add a book and track your progress to finishing reading it. If you want to remove any, simply click on remove button and it is deleted.
+
+# Screenshot
+![Screenshot (34)](https://user-images.githubusercontent.com/90222110/155992775-ca8a575a-a107-478e-a698-cdb9021667bd.png)
+
+
+# Live Demo
+Not yet deplyed
+
+# Built with
+- React.js
+- Redux
+- HTML
+- CSS
+
+# Other tools
+- Webpack and Babel compiler
+
+
+# Usage
+
+## installation
+
+This project has a lot of dependencies. In order to run on your machine very well, you need to install them by typing in the terminal
+
```

Comment:
- [ ] Let's try to be detailed in our documentation as it can be of great benefit to users. In order to run this installation locally, users need to have cloned the repo. Kindly add instructions on how to clone the repo.